src/query/sql/src/planner/plans/window.rs

```diff
@@ -90,6 +90,25 @@ impl Window {
 
         Ok(used_columns)
     }
+
+    // group all used columns in window function
```

Comment:
What's the difference with the above `fn used_columns`?

Reply:
I think this refers to the method call in `RulePushDownFilterAggregate`. However, `Window` already has a method `used_columns`.